projects/go-style-guide/src/app/features/ui-kit/components/toast-docs/toast-docs.component.html

```diff
@@ -143,4 +143,39 @@ <h4 class="go-heading-4">app.component.html</h4>
       <go-button (handleClick)="sendToast()">Make Toast</go-button>
     </ng-container>
   </go-card>
+  <go-card class="go-column go-column--100">
+    <ng-container go-card-header>
+      <h2 class="go-heading-2">Button Example</h2>
+    </ng-container>
+    <ng-container go-card-content>
+        <p class="go-body-copy">
+        Toaster actions are a set of buttons and other tools meant to perform actions on the Toaster.
```

Comment:
Can this be changed to:
`Toast actions are a set of buttons and other tools meant to perform actions on the Toast.`